arcgis-ios-sdk-samples/Geometry/Nearest vertex/README.md

```diff
@@ -23,8 +23,14 @@ Tap anywhere on the map. An orange cross will show at that location. A blue circ
 ## Relevant API
 
 * AGSGeometry
-* AGSGeometryEngine
 * AGSProximityResult
+* class AGSGeometryEngine.nearestCoordinate(in:to:)
+* class AGSGeometryEngine.nearestVertex(in:to:)
+* class AGSGeometryEngine.normalizeCentralMeridian(of:)
+
+## Additional information
+
+The value of `AGSProximityResult.distance` is planar (Euclidean) distance. Planar distances are only accurate for geometries that have a defined projected coordinate system, which maintain the desired level of accuracy. The example polygon in this sample is defined in California State Plane Coordinate System - Zone 5 (WKID 2229), which maintains accuracy near Southern California. Accuracy declines outside the state plane zone.
```

Comment:
I adopted a few phrasing from this doc: https://desktop.arcgis.com/en/arcmap/10.3/guide-books/map-projections/state-plane-coordinate-system.htm